src/Form/SettingsForm.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\y_lb\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Settings Form for Small Y.
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'ws_admin_settings';
```

Comment:
Maybe it is better to be consistent with the config name and use something like `y_lb_settings_form`?